cypress/support/page-objects/storefront/quick-order/storefront-quick-order-page.ts

```diff
@@ -0,0 +1,82 @@
+import { AbstractPage } from '../../abstract-page'
+
+export class StorefrontQuickOrderPage extends AbstractPage {
+  protected PAGE_URL = Cypress.env('STOREFRONT_URL') + '/en/quick-order'
+
+  getQuickOrderForm = (): Cypress.Chainable => {
+    return cy.get('[data-qa="component quick-order-form"]')
+  }
+
+  getQuickOrderRows = (): Cypress.Chainable => {
+    return this.getQuickOrderForm().find(
+      '[data-qa="component quick-order-row"]'
+    )
+  }
+
+  getProductSearchAutocompleteFields = (): Cypress.Chainable => {
+    return cy.get('[data-qa="component product-search-autocomplete-form"]')
+  }
+
+  searchProduct = (
+    fieldIndex: number,
+    skuOrName: string
+  ): Cypress.Chainable => {
+    return this.getProductSearchAutocompleteFields()
+      .eq(fieldIndex)
+      .type(skuOrName)
+  }
+
+  applySuggestedProduct = (skuOrName: string): Cypress.Chainable => {
+    return cy
+      .get('[data-qa="component products-list"]')
```

Comment:
1. Can we move `get('[data-qa="component products-list"]` to a separate getter methods, like `getSuggestedProductsList`?
2. Will it work if I perform a product search more than 1 time? Like in row 1 I type smth and get the results. Then I do search in row 2.  I guess it can be improved with `.filter(':visible')`
3. What if suggestions list will contain 2 and more items that match skuOrName?

Reply:
1. Good point! Moved to a separate method
2. Yes, it would, but with your suggestion looks more stable. I added it.
3. According to a default behaviour of Cypress, it selects the first suitable option